### PR TITLE
Add IRC webclient link to get involved

### PR DESCRIPTION
**Pull request by rpannek**
Not everybody has a IRC client set up already, this link helps them
to join the IRC chanel from their browser without the need to install
anything.

### CODE REVIEW

File: get-involved.md [hunk 1]
```diff
@@ -9,8 +9,13 @@ If you're interested to get involved in the developement then there are many way
 ## IRC
 
 The best way is to join our IRC chanel #PELUX on freenode, where most of the developers hand out.
+Join the chanel in a webbrowser:
```
Comment (GordanM) [inline get-involved.md]: Nitpick 1: channel with two "n"s.  
Nitpick 2: Web browser should be written as two words.

Comment (rpannek) [inline get-involved.md]: fixed

File: get-involved.md [hunk 2]
```diff
@@ -8,9 +8,14 @@ If you're interested to get involved in the developement then there are many way
 
 ## IRC
 
-The best way is to join our IRC chanel #PELUX on freenode, where most of the developers hand out.
+The best way is to join our IRC channel #PELUX on freenode, where most of the developers hand out.
+Join the chanel in a web browser:
```
Comment (GordanM) [inline get-involved.md]: The channel should have two "n"s - i.e. channel.

Comment (DunderRoffe) [inline get-involved.md]: Fix the same thing in the commit message as well :)

File: get-involved.md [hunk 3]
```diff
@@ -8,9 +8,14 @@ If you're interested to get involved in the developement then there are many way
 
 ## IRC
 
-The best way is to join our IRC chanel #PELUX on freenode, where most of the developers hand out.
+The best way is to join our IRC channel #PELUX on freenode, where most of the developers hand out.
```
Comment (jeremiah) [inline get-involved.md]: I have another PR that changes "hand" to "hang" and some other minor things: https://github.com/Pelagicore/pelux.io/pull/3

Comment (rpannek) [inline get-involved.md]: I merged your PR and rebased mine on top of it.

**Comment by rpannek** [comment]
Ok, I hope I got those changes right now.

**Pull request merged**